hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieCopyOnWriteTableInputFormat.java

```diff
@@ -161,6 +173,15 @@ protected FileSplit makeSplit(Path file, long start, long length,
     return returns.toArray(new FileStatus[0]);
   }
 
+  @Override
+  public RecordReader<NullWritable, ArrayWritable> getRecordReader(InputSplit split, JobConf job, Reporter reporter) throws IOException {
+    throw new UnsupportedEncodingException("not implemented");
```

Comment:
Why are we throwing. `EncodingException`. Can we throw `HoodieUnsupported..`

Reply:
Good catch! Just a typo, will address in follow-up